New translations structure

dennisreimann opened the pull request:
Closes #105.

dennisreimann committed:
Bump completeness threshold to 100%

dennisreimann committed:
Remove old php translator script

dennisreimann committed:
Update templates for new translation structure

Closes #105.

pavlenex commented on source/src/vtt/video_tmpl.vtt:
```diff
@@ -2,84 +2,84 @@ WEBVTT - This file has no cues.
 
```
Do not change source/src/vtt/video_tmpl.vtt, it's unrelated to the change we did it's a static translation that won't change, so best to leave it legacy way that it is.

dennisreimann replied:
The content/output does not change with these modifications, but I had to update the template with the new variable format.

pavlenex replied:
If you tested that video translation is not broken, okay. We have not updated the video.json https://www.transifex.com/btcpayserver/btcpayserver-website/video_en_json/ to reflect the new format, best to leave it the way it was.

dennisreimann replied:
The output is just the same, but [the variable format changed](https://github.com/btcpayserver/btcpayserver.org/pull/106/commits/cd84959c8436c0a3ebd14f6dffa81ff38f3cd4f3#diff-4e3ef10b64d9e4e436aec35246cc1c1eR89) and I had to reflect that here, so that we don't use different substitution mechanisms in the html and vtt templates (because they use the same function). 

You can compare the output by building the master branch, copying the `dist` forlder to `dist-old` and then switching to this PR branch and build anew.

dennisreimann committed:
Set completeness threshold to 90%

That's about five new strings that can be added before a language that was complete before would drop out.

dennisreimann committed:
Fix theme vars
